vendor.mod

```diff
@@ -131,7 +131,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
-	github.com/cilium/ebpf v0.9.1 // indirect
+	github.com/cilium/ebpf v0.12.3 // indirect
```

Comment:
It looks like this is an indirect dependency, and used by `github.com/containerd/cgroups/v3`. Wondering if it should be updated there first (as well)

Reply:
Okay, I'll go take a look.